src/gui/mzroll/pollyelmaveninterface.h

```diff
@@ -142,6 +149,9 @@ class PollyElmavenInterfaceDialog : public QDialog, public Ui_PollyElmavenInterf
                  * 5. Populate all combo boxes on this UI, with the data obtained from above steps..
                  */
                 QVariantMap startup_data_load();
```

Comment:
This returns QVariantMap but it is not used anywhere. 
Can be changed to `void`

Thought it would be good if we keep cleaning things like these in parallel

Reply:
projectnames_id was made a global parameter recently so yes. Can be changed to void. Will do it while refactoring.